src/sentry/tasks/commits.py

```diff
@@ -220,3 +223,17 @@ def fetch_commits(release_id, user_id, refs, prev_release_id=None, **kwargs):
 
 def is_integration_provider(provider):
     return provider and provider.startswith("integrations:")
+
+
+def get_owner_emails(user, orgId):
```

Comment:
@NisanthanNanthakumar This function probably should be named something like `get_emails_for_user_or_org` since it's not going to return the owner email if the user is just a user.